README.md

```diff
@@ -108,160 +111,178 @@ See [CoaL Setup](./docs/developer-guide/coal-setup.md) for a thorough walkthroug
+
+As mentioned previously, see [CoaL Setup](./docs/developer-guide/coal-setup.md)
+for a thorough walkthrough including enabling provisioning on the headnode for
+development.
 
 
 ### Installing SDC on a Physical Server
 
-A SmartDataCenter server runs SmartOS, which is a "live image". That means that
-it boots from a USB key. Installing SDC involves writing a "USB" build to
+A SmartDataCenter server runs SmartOS which is a live image. This means that
+it boots from a USB flash drive (key).
 a physical USB key, inserting the key and booting the server from that key.
 To install SDC, first obtain the latest release USB build.
 
+
 #### Hardware
 
 For SDC development only, the minimum server hardware is:
 
 - 8 GB USB flash drive
-- Intel Processors with VT-x and EPT support (all Xeon since Nehalem).
+- Intel Processors with VT-x and EPT support (all Xeon since Nehalem)
 - 16 GB RAM
-- 6 GB available storage. Hardware RAID is not recommended.
-  SDC will lay down a ZFS ZPOOL across all available disks on install.
-  You'll want much more storage if you're working with images and instances.
+- 6 GB available storage
+
+Hardware RAID is not recommended. SDC will lay down a ZFS ZPOOL across all
+available disks on install. You'll want much more storage if you're working with
+images and instances.
 
 If setting up a SmartDataCenter pilot then you'll want to review
-the [SDC7 Installation Prerequisites](https://docs.joyent.com/sdc7/sdc7-installation-prerequisites)
+the [Minimum Requirements](https://docs.joyent.com/sdc7/sdc7-minimium-requirements)
+and [Installation Prerequisites](https://docs.joyent.com/sdc7/sdc7-installation-prerequisites)
 which include IPMI and at least 10 gigabit Ethernet. The supported hardware
 components for SmartOS are described in the [SmartOS Hardware Requirements](http://wiki.smartos.org/display/DOC/Hardware+Requirements).
 Joyent certified hardware for SmartDataCenter are all in
 the [Joyent Manufacturing Database](http://eng.joyent.com/manufacturing/).
 
 
-#### USB Key
+#### Install
 
-Download the USB key image:
+To install SDC, first download the latest release image:
 
 ```bash
 curl -C - -O https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/usb-latest.tgz
 ```
 
-#### Install
-
-Once you have downloaded an image, you will need to
+Once you have downloaded the latest release image, you will need to
 [write it to a USB key](https://docs.joyent.com/sdc7/installing-sdc7/creating-a-usb-key-from-a-release-tarball),
-boot the machine with it, and follow the install prompts. See the
-[installing SDC 7](https://docs.joyent.com/sdc7/installing-sdc7) and
-[install checklist](https://docs.joyent.com/sdc7/installing-sdc7/install-checklist)
-documents for information.
+boot the headnode server using the USB key, and follow the install prompts. See
+the the Joyent customer documentation "[installing SDC 7](https://docs.joyent.com/sdc7/installing-sdc7)"
```

Comment:
Again, why is this specifically Joyent customer documentation. The long term goal is not to treat this as separate, even though it is at the moment.